perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -362,6 +374,10 @@ sub inspect : Private {
         }
 
         if ($valid) {
+            if ( $reputation_change != 0 && !$problem->get_extra_metadata('user_reputation_set') ) {
+                $problem->user->update_reputation($reputation_change);
+                $problem->set_extra_metadata( user_reputation_set => 1)
```

Comment:
(After the changes to remove negative_states), do we mind if somehow the report got unhidden/uninspected and done again?